bodhi/tests/server/test_alembic.py

```diff
@@ -22,14 +22,15 @@
 
 import os
 import subprocess
-import unittest
+
+from bodhi.tests.server.base import BasePyTestCase
 
 
 REPO_PATH = os.path.abspath(
     os.path.join(os.path.dirname(__file__), '..', '..', '..'))
 
 
-class TestAlembic(unittest.TestCase):
+class TestAlembic(BasePyTestCase):
```

Comment:
This wasn't derived from `BaseTestCase` before, i.e. it doesn't need the test database etc. and the incurred overhead which comes with `BasePyTestCase`. Just don't inherit from `unittest.TestCase`, i.e. make it:

````
class TestAlembic:
    ...
````